Resources/public/ts/StingerSoftAggrid.ts

```diff
@@ -110,7 +110,8 @@ export class StingerSoftAggrid {
         return jQuery.getJSON(this.options.stinger.ajaxUrl, {
             'agGrid': {
                 'gridId': '#' + that.gridId
-            }
+            },
+            'gridHelper': this.options['gridHelper']
```

Comment:
Please call the vairable something like "additionalAjaxRequestBody" and merge it into the existing data passed to the getJson call